Formula/ford.rb

```diff
@@ -13,52 +13,63 @@ class Ford < Formula
     sha256 "0b513d22c17e06f4d7021d9f69a0254a33c24e5197b02224280e275af0599971" => :mavericks
   end
 
+  option "without-lxml", "Do not install lxml to improve the speed of search  database generation"
+
   depends_on "graphviz"
   depends_on :python if MacOS.version <= :snow_leopard
 
+  if build.with? "lxml"
+    resource "lxml" do
+      url "https://pypi.python.org/packages/11/1b/fe6904151b37a0d6da6e60c13583945f8ce3eae8ebd0ec763ce546358947/lxml-3.6.0.tar.gz"
+      sha256 "9c74ca28a7f0c30dca8872281b3c47705e21217c8bc63912d95c9e2a7cac6bdf"
+    end
+  end
```

Comment:
Ping. 😸 Per https://github.com/Homebrew/homebrew-core/pull/1914#issuecomment-225459562 I believe this is the only thing that stops us from merging this …